src/widgets/Catalog/filters/ui.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
+import styled from 'styled-components';
```

Comment:
Возможно у тебя отключены какие-то проверки, но IDE должна ругаться на такой импорт.
ESLint: Using exported name 'styled' as identifier for default export. (import/no-named-as-default)
```suggestion
import { styled } from 'styled-components';
```

Reply:
У нас настройками tsconfig разрешены такие импорты.
"esModuleInterop": true

А тут поясняется почему это хорошо:
https://habr.com/ru/companies/otus/articles/507104/

Reply:
Понятно. Я не уверен в том, что у нас идеально настроенный tsconfig, т.к. опыта настройки у меня мало. У меня в IDE вот такой варнинг высвечивается:
ESLint: Using exported name 'styled' as identifier for default export. (import/no-named-as-default)